sdk/storage/azure-storage-blob/tests/test_container.py

```diff
@@ -71,6 +72,14 @@ def _create_container(self, prefix=TEST_CONTAINER_PREFIX):
             pass
         return container
 
+    def _generate_oauth_token(self):
```

Comment:
Is this being used? Also the testcase.py should have the same method AFAIK.

Reply:
this is for async version. the one in testcase is for sync